document-generator-prosecution/src/main/java/uk/gov/companieshouse/document/generator/prosecution/exception/ProsecutionServiceException.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.companieshouse.document.generator.prosecution.exception;
+
+import uk.gov.companieshouse.document.generator.prosecution.service.ProsecutionService;
+
+/**
+ * Thrown whenever an error occurs in the {@link ProsecutionService}
+ */
+public class ProsecutionServiceException extends Exception{
+    public ProsecutionServiceException(String message) {
+        super(message);
+    }
+
+    public ProsecutionServiceException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public ProsecutionServiceException(String message, Throwable cause, boolean enableSuppression,
+            boolean writableStackTrace) {
+        super(message, cause, enableSuppression, writableStackTrace);
```

Comment:
Why is this constructor required?

Reply:
It's not, I have removed the unnecessary constructors